### DISCUSSION TITLE
Issue with Facebook URL Accessibility in .NET MAUI WebView on Windows 10

### DISCUSSION THREAD

**Discussion by PrabhuRout007** [MicrosoftEdge/WebView2Feedback#4295] [Q&A] [answered]
I am currently developing a .NET MAUI application using .NET 7 for Windows desktop application, and I have encountered an issue with the WebView control on Windows 10 when attempting to load Facebook URLs. The specific version of WebView Runtime in use is 120.0.2210.121.
However, on Windows 10, attempts to access Facebook URLs within the WebView Runtime are unsuccessful. This issue is hindering the integration of Facebook Graph API for essential functionalities such as login and other activities.

**Comment by victorhuangwq** [comment, marked as answer]
This is a known issue and it's currently tracked here: https://github.com/MicrosoftEdge/WebView2Feedback/issues/4281

> The issue is fixed in Edge Canary 122.0.2330.0+. Could you please test with [Canary channel](https://learn.microsoft.com/en-us/microsoft-edge/webview2/how-to/set-preview-channel) and validate the fix. Thanks!

> We are working on backporting the fix to Edge Stable and should be available soon. I'll post the updates in the thread.
